CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+
+Diversity and inclusion make the Docker community strong. We encourage participation from the most varied and diverse backgrounds possible and want to be very clear about where we stand.
+
+Our goal is to maintain a safe, helpful and friendly Docker community for everyone, regardless of experience, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other protected categories under applicable law.
+
+## Expected Behavior
+
+- Be professional.
+- Be responsible.
+- Be welcoming.
+- Be kind.
+- Be respectful of other viewpoints and ideas.
+- Be supportive and look out for each other.
+
+## Unacceptable Behavior
+
+Harassment may include but not limited to the following:
+
+- Offensive, inappropriate, or unwanted comments related to gender, gender identity or expression, sexual orientation, disability, physical appearance, body size, race, ethnicity, national origin, religion, or age, or other protected categories under applicable law.
+- Visual harassment eg. sexual imagery or use of sexual language at Docker community events
+- Disrespect towards differences of opinion
+- Deliberate intimidation, stalking, harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate or unwanted physical contact
+- Intimidation or bullying (online or in-person)
+- Unwelcome sexual attention
+
+## Reporting and Enforcement
+
+- If you are the subject of, or witness to any violations of this Code of Conduct, please contact us by submitting an [incident report](http://noop.incidentreportform.app), or email noopcon@rearc.io
+
+Portions derived from the [Slack Developer Community Code of Conduct](https://api.slack.com/docs/community-code-of-conduct), [The Ada Initiative](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/), [geekfeminism.org](https://geekfeminism.org/about/code-of-conduct/) and [Drupal Events Code of Conduct](https://events.drupal.org/dublin2016/code-conduct)
+
+This work is licensed under a Creative Commons Attribution 3.0 Unported License For attribution requirements:
+
+"@Docker Code of Conduct" © 2016 Docker, Inc, used under a Creative Commons Attribution Unported license: [http://creativecommons.org/licenses/by/3.0/](http://creativecommons.org/licenses/by/3.0/)
```

Comment:
We should plan to go through and update this `CODE-OF-CONDUCT.md` doc to be customized with Noop Inc references before this PR is merged into `main`

Reply:
> 



> @apsamuel
> 
> > ...**also confirmed the currently supported feature set doesn't provide for user-added options in select menus.**
> 
> I think that's a pretty critical feature which we need to find a way to accommodate. The alternative is one of us needs to remember to regularly audit/adjust the hardwired options in this repo's issue templates which isn't ideal.

Switched to text area for version, removed required flag.